concordium-consensus/src/Concordium/ProtocolUpdate.hs

```diff
@@ -33,7 +33,7 @@ checkUpdate = case protocolVersion @pv of
     SP1 -> fmap UpdateP1 . P1.checkUpdate
     SP2 -> fmap UpdateP2 . P2.checkUpdate
     SP3 -> fmap UpdateP3 . P3.checkUpdate
-    SP4 -> const $ Left "Unsupported update."
+    _ -> const $ Left "Unsupported update."
```

Comment:
Is it intensional not to support updating yet?

Reply:
I have added an update to P5 now.